test/geocel/vg/Vecgeom.test.cc

```diff
@@ -1278,7 +1251,7 @@ TEST_F(FourLevelsGeantTest, tracking)
     {
         // Formerly in linear propagator test, used to fail
         SCOPED_TRACE("From just outside world");
-        auto result = this->track({-24, 10, 10}, {1, 0, 0});
+        auto result = this->track({-24.1, 10, 10}, {1, 0, 0});
```

Comment:
Looks like this test is actually a duplicate of the one above: I'm just going to delete this one.

Reply:
The difference is the Geant vs. no-Geant, right?  Converted geometry vs. VGDML-based?
I thought you wanted to keep because of that.  I am fine with removing one of them.

Reply:
I meant it's a duplicate of `From outside edge` a few lines above 😅